test/smoke/testApps/SpringBootAuto/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/SpringBootAutoTest.java

```diff
@@ -1,13 +1,29 @@
 package com.microsoft.applicationinsights.smoketest;
 
+import java.util.ArrayList;
+import java.util.List;
+
+import com.microsoft.applicationinsights.internal.schemav2.Data;
+import com.microsoft.applicationinsights.internal.schemav2.Envelope;
+import com.microsoft.applicationinsights.internal.schemav2.RequestData;
 import org.junit.Test;
 
+import static org.junit.Assert.assertEquals;
+
 @UseAgent
 public class SpringBootAutoTest extends AiSmokeTest {
 
     @Test
     @TargetUri("/test")
     public void doMostBasicTest() throws Exception {
-        mockedIngestion.waitForItems("RequestData", 1);
+        List<Envelope> rdList = mockedIngestion.getItemsEnvelopeDataType("RequestData");
+        List<Envelope> httpRequestList = new ArrayList<>();
+        for (Envelope envelope : rdList) {
+            RequestData rd = (RequestData) ((Data) envelope.getData()).getBaseData();
+            if (rd.getName().equals("GET /test")) {
+                httpRequestList.add(envelope);
+            }
+        }
+        assertEquals(1, httpRequestList.size());
```

Comment:
this is more consistent with other smoke test validation:

```suggestion
        List<Envelope> rdList = mockedIngestion.waitForItems("RequestData", 1);
        Envelope rdEnvelope = rdList.get(0);
        RequestData rd = (RequestData) ((Data) rdEnvelope.getData()).getBaseData();

        assertEquals("GET /test", rd.getName());
```

Reply:
waitForItems for 1 doesn't work anymore.. Scheduler is inside SpringBootAuto.

Reply:
this is not scheduler test tho

Reply:
yes, it's not, but scheduler is enabled inside SprintBootAuto.  RequestData can be scheduler request or http request.